src/containers/FavoritesContainer.ts

```diff
@@ -0,0 +1,62 @@
+import { connect } from 'react-redux';
+import { AppState } from '../reducers';
+import { StateProps, DispatchProps, YourFeed } from '../components/YourFeed';
+import { AsyncActions, Actions } from '../actions/Actions';
+import { Post } from '../models/Post';
+import { Feed } from '../models/Feed';
+import { List } from 'immutable';
+
+const isPostFromFavoriteFeed = (post: Post, feeds: List<Feed>): boolean => {
+    return feeds.find(feed => {
+        return feed != null && post.author != null &&
+            feed.feedUrl === post.author.uri &&
+            feed.favorite === true;
+    }) != null;
+};
+
+const mapStateToProps = (state: AppState, ownProps): StateProps => {
+    const posts = state.rssPosts
+        .filter(post => post != null && isPostFromFavoriteFeed(post, state.feeds))
```

Comment:
It would be better to create a variable that holds the favorite feeds and then pass that list to isPostFromFavorites.